src/components/Login.jsx

```diff
@@ -50,7 +50,7 @@ function Login() {
               <form 
                 className="space-y-4 md:space-y-6" 
                 onSubmit={handleSubmit}
-                action="#">
```

Comment:
I think this action="#" might be necessary. Otherwise this submit button: https://github.com/sebasutp/screenshot_frontend/blob/89a6f46b67ac36457aeb1266fe17a57d1f2672ef/src/components/Login.jsx#L95-L98

Might create a default "post" action.

Reply:
This property `action="#"` on the form is not required, as it was not needed before.